app/controllers/home_controller.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
 class HomeController < ApplicationController
-  def index; end
+  def index
+    men_cat_id = Category.find_by(name: 'Men')
```

Comment:
Jak to można inaczej rozwiązać:

dodanie kolumny bool (flagi) `main_men_category: true` do modelu category
i wtedy `men_cat_id = Category.find_by(main_men_category: true)`

Reply:
Zasada - kod niezależny od danych z bazy